lta/rest_server.py

```diff
@@ -17,7 +17,8 @@
 from pymongo import MongoClient
 from rest_tools.client import json_decode  # type: ignore
 from rest_tools.server import authenticated, catch_error, RestHandler, RestHandlerSetup, RestServer  # type: ignore
-import tornado.web  # type: ignore
+from str2bool import str2bool  # type: ignore
```

Comment:
I object to a dependency this simple, just on principle.

Reply:
But have we truly lived until we experience LTA stop working when a developer unpublishes the left_pad package? :-)

Reply:
Done: https://github.com/WIPACrepo/lta/pull/25/commits/76fc3e039b8ace59f905d1b1e328cd1cccc6129f